src/pages/test-center.page.js

```diff
@@ -56,15 +56,28 @@ class TestCenterPage extends Component {
     window.addEventListener('resize', debounce(this.onWindowResize, 400));
     const sessionId = get(this, ['context', 'appState', 'sessionId']);
     this.isLoggedIn = sessionId ? true : false;
-    const facility = await this.facilityService.getFacility(this.id)
+    const isActualID = !isNaN(Number(this.id));
```

Comment:
Number converts non-number values to 0, so this won't check anything. isNaN on number strings should do what you want 
`!isNaN('12345') = true
!isNaN('hello') = false`

Reply:
![image](https://user-images.githubusercontent.com/3998637/83316453-d459c800-a1f3-11ea-9208-dea7c92b5b1a.png)

i think this works as expected